vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -4069,6 +4445,126 @@ unit:DeciM
+  qudt:hasQuantityKind quantitykind:VolumeFlowRate ;
+  qudt:iec61360Code "0112/2///62720#UAA416" ;
+  qudt:uneceCommonCode "E92" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciM3 PER HR" ;
+  skos:prefLabel "decimetre cubed per hour" ;
+.
+unit:DeciM3-PER-M3
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-03 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "volume ratio consisting of the 0,001-fold of the power of the SI base unit metre with the exponent 3 divided by the power of the SI base unit metre with the exponent 3" ;
+  qudt:hasQuantityKind quantitykind:VolumeFraction ;
+  qudt:iec61360Code "0112/2///62720#UAA417" ;
+  qudt:uneceCommonCode "J91" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciM3 PER M3" ;
+  skos:prefLabel "decimetre cubed per metre cubed" ;
+.
+unit:DeciM3-PER-MIN
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1.666667e-05 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,001-fold of the power of the SI base unit metre with the exponent 3 divided by the unit for time minute" ;
+  qudt:hasQuantityKind quantitykind:VolumeFlowRate ;
+  qudt:iec61360Code "0112/2///62720#UAA418" ;
+  qudt:uneceCommonCode "J92" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciM3 PER MIN" ;
+  skos:prefLabel "decimetre cubed per minute" ;
+.
+unit:DeciM3-PER-MOL
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-03 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,001-fold of the power of the SI base unit metre with the exponent 3 divided by the SI base unit mol" ;
+  qudt:hasQuantityKind quantitykind:MolarVolume ;
+  qudt:iec61360Code "0112/2///62720#UAA419" ;
+  qudt:uneceCommonCode "A37" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciM3 PER MOL" ;
+  skos:prefLabel "decimetre cubed per mole" ;
+.
+unit:DeciM3-PER-SEC
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-03 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,001-fold of the power of the SI base unit metre with the exponent 3 divided by the unit for time second" ;
+  qudt:hasQuantityKind quantitykind:VolumeFlowRate ;
+  qudt:iec61360Code "0112/2///62720#UAA420" ;
+  qudt:uneceCommonCode "J93" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciM3 PER SEC" ;
+  skos:prefLabel "decimetre cubed per second" ;
+.
+unit:DeciN-M
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-01 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,1-fold of the product of the derived SI unit joule and the SI base unit metre" ;
+  qudt:hasQuantityKind quantitykind:Torque ;
+  qudt:iec61360Code "0112/2///62720#UAB084" ;
+  qudt:uneceCommonCode "DN" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciN M" ;
+  skos:prefLabel "decinewton metre" ;
+.
+unit:DeciTON
+  a qudt:Unit ;
+  qudt:conversionMultiplier 9.071847e+01 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "100-fold of the SI base unit kilogram" ;
```

Comment:
This seems an odd description, since this is not equal to 100 kilograms.

Reply:
fixed